sensor_msgs/Image ros_image
